src/SelectMenu/SelectMenuFilter.js

```diff
@@ -28,8 +31,14 @@ function SelectMenuFilter({theme, value, ...rest}) {
       inputRef.current.focus()
     }
   }, [open])
+
+  let sxProps = {}
+  if (sx) {
+    sxProps = {sx}
+  }
```

Comment:
What's this for?

Reply:
Good catch, situations like this came up a few times and I experimented with a few patterns. I'll go back and take a second look.